### DISCUSSION TITLE
@entangle does not update array property on the php side

### DISCUSSION THREAD

**Discussion by MichaelBrauner** [livewire/livewire#1750] [4. General]
I write an inline select field (like select2) and wanted to use @entangle. 

The initial value is set fine inside alpinejs but when I add or remove selections no ajax requests are fireing off and also the 
updatedValue() function is not triggered. 

Is that a bug? Or do I miss something important?

I created a short [LaravelPlayground](https://laravelplayground.com/#/snippets/53e0d20d-5624-4691-9b37-d81446d25fec) for that. The @entangle is on line 31 bound to the tags value inside the hello-world.blade.php. 

Hope someone can help with that.

**Comment by isaackearl** [comment]
If you recently upgraded livewire locally, try and make sure that you published the latest assets, if you don't then the fix to the array problem won't work.  For me #1682 fixed my issue and I've been able to entangle simple arrays no problem.  If that doesn't work then maybe a bit later I'll have some time to try out your issue locally for me and see if I can compare it to what I'm doing.

As a side note, I don't think laravelplayground has upgraded to the latest livewire yet.  I could be wrong though.

**Reply by MichaelBrauner** [reply]
Ok, thank you. 

I just did a `php artisan view:clear` and then `php artisan livewire:publish --assets`.
I checked the livewire version with `composer show livewire/livewire` and it is `versions : * v2.2.9`.

Now I tried your simple [example](https://laravelplayground.com/#/snippets/38ba0f3a-9f43-4d41-9be1-6661330f1e93) inside my lokal setup. Just the 'Java Script List' is updated and not the 'Server rendered List:' when I click on 'Add Motorcycle'.

I don't know - but thats weird that is works just fine on your machine.
